marconi-chain-index/src/Marconi/ChainIndex/Indexers/UtxoQuery.hs

```diff
@@ -282,9 +290,14 @@ baseQuery =
   LEFT JOIN blockInfo.blockInfo spentBlockInfo ON spentBlockInfo.slotNo = futureSpent.slotNo
   |]
 
+-- Core.Queryable m BlockInfo (Core.EventsMatchingQuery BlockInfo) Core.SQLiteIndexer
```

Comment:
Is there a reason for this comment to be there?

Reply:
That will have been a reminder to myself that I forgot to remove!